js/symbol/glyph_source.js

```diff
@@ -28,7 +28,7 @@ GlyphSource.prototype.getSimpleGlyphs = function(fontstack, glyphIDs, uid, callb
         this.stacks[fontstack] = {};
     }
     if (this.atlases[fontstack] === undefined) {
-        this.atlases[fontstack] = new GlyphAtlas(128, 128);
```

Comment:
@bhousel It's still 128! I moved this constant inside `GlyphAtlas` because there are some tricky dependencies between the default size, growth factor, and max size. Having them all in one place makes that clearer.